examples/arkane/species/Toulene_Hindered_Rotor_SemiClassicalND/output.py

```diff
@@ -0,0 +1,74 @@
+# Coordinates for Toluene in Input Orientation (angstroms):
```

Comment:
No need to commit output files (in. chemkin files and co) for examples, unless we have a good reason to (?)

Reply:
let me know the status of this comment

Reply:
Sorry, missed this the first time through, should be fixed now.